docsets/WPILib_Cpp/docset.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "WPILib_Cpp",
+    "version": "2020.2.2",
+    "archive": "WPILib_Cpp.tgz",
```

Comment:
The archive at this path contains the WPILib Java docset, not the C++ one

Reply:
Whoops.  Guess I messed up one of my moves...
Good catch: thanks